content/blog/azure-v6-release/index.md

```diff
@@ -14,8 +14,7 @@ tags:
 The latest major release of the Azure Classic Pulumi Provider is available now!
 Our 6.0 release contains the [latest upstream changes](https://github.com/hashicorp/terraform-provider-azurerm/releases/tag/v4.0.1) to keep you up-to-date along with the latest features and improvements from Pulumi.
 
-The Pulumi Azure Native provider is our recommended approach for building on Azure which is generated directly from the API specifications so has amazing resource coverage.
-The Azure Classic provider is derived from the upstream Terraform AzureRM provider and can also be used to provision most resources on Azure, and can be used in conjunction with the Native Provider where needed as a useful alternative.
+The Azure Classic provider is derived from the upstream Terraform AzureRM provider and can also be used to provision most resources on Azure, and can be used to complement the [Pulumi Azure Native provider](https://www.pulumi.com/registry/packages/azure-native/) - our recommended provider which has amazing resource coverage due to being derived directly from the official specifications.
```

Comment:
sorry one more nit: you've now got a run on sentence here "... __and__ can also be used to provision most resources on Azure, __and__ can be used to complement..."

Reply:
Oof, yes. Broke it up 👍